docs/source/quickstart/99_FAQ.md

```diff
@@ -149,8 +149,8 @@ This one will remove the running resources, mostly:
 It's probably the most used, since it allows a faster re-deploy later.
 
 ```Bash
-[laptop]$ ansible-playbook [-i custom/inventory.yml] \
-    [-e cifmw_target_host=hypervisor-1] \
+[laptop]$ ansible-playbook -i custom/inventory.yml \
+    -e cifmw_target_host=hypervisor-1 \
```

Comment:
no - because that one is optional. This is standard syntax for optional parameters.